docs/source-pytorch/fabric/fabric.rst

```diff
@@ -589,3 +613,22 @@ For single-device strategies, it is a no-op. There are strategies that don't sup
 - xla
 
 For these, the context manager falls back to a no-op and emits a warning.
+
+
+log and log_dict
+================
+
+These methods allows you to send scalar metrics to a logger registered in Fabric.
+
+.. code-block:: python
+
+    # Set the logger in Fabric
+    fabric = Fabric(loggers=TensorBoardLogger(...))
+
+    # Anywhere in your training loop or model:
+    fabric.log("loss", loss)
+
+    # Or send multiple metrics at once:
+    fabric.log_dict({"loss": loss, "accuracy": acc})
+
+If no loggers are given to Fabric (default), `log` and `log_dict` won't do anything.
```

Comment:
Can you add two lines of pseudocode showing that `fabric.log` basically routes to `logger.log_*`? This way it's clear to the users that they could also use the logger themselves.

Reply:
done